product-scenarios/code-coverage/pom.xml

```diff
@@ -0,0 +1,130 @@
+                                    <target xmlns:jacoco="antlib:org.jacoco.ant">
+                                        <taskdef uri="antlib:org.jacoco.ant" resource="org/jacoco/ant/antlib.xml">
+                                            <classpath path="${project.build.directory}" />
+                                        </taskdef>
+                                        <jacoco:report>
+                                            <executiondata>
+                                                <fileset dir="../..">
+                                                    <include name="**/code-coverage/*/instance1/*/jacoco.exec" />
+                                                </fileset>
+                                                <fileset dir="../..">
+                                                    <include name="**/code-coverage/*/instance2/*/jacoco.exec" />
+                                                </fileset>
+                                            </executiondata>
+                                            <structure name="Scenario Coverage Report">
+                                                <classfiles>
+                                                    <fileset dir="../..">
+                                                        <include name="**/code-coverage/*/instance1/*/classes/**" />
+                                                    </fileset>
+                                                    <fileset dir="../..">
+                                                        <include name="**/code-coverage/*/instance2/*/classes/**" />
+                                                    </fileset>
+                                                </classfiles>
+                                            </structure>
+                                            <html destdir="${project.build.directory}/scenario-code-coverage/jacoco" />
+                                            <xml destfile="${project.build.directory}/scenario-code-coverage/jacoco/coverage-report.xml"/>
+                                            <csv destfile="${project.build.directory}/scenario-code-coverage/jacoco/coverage-report.csv"/>
+                                        </jacoco:report>
+                                    </target>
+                                </configuration>
+                            </execution>
+                        </executions>
+                        <dependencies>
+                            <dependency>
+                                <groupId>org.jacoco</groupId>
+                                <artifactId>org.jacoco.ant</artifactId>
+                                <version>${jacoco.version}</version>
+                            </dependency>
+                        </dependencies>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+    <properties>
+        <jacoco.version>0.7.9</jacoco.version>
```

Comment:
move this property to product-scenarios pom

Reply:
Fixed. Please refer latest commit.